SwiftyStoreKit/SwiftyStoreKit.swift

```diff
@@ -118,17 +118,10 @@ public class SwiftyStoreKit {
         return RestoreResults(restoredPurchases: restoredPurchases, restoreFailedPurchases: restoreFailedPurchases)
     }
     
-    #if os(watchOS)
```

Comment:
Shouldn't this have the addition of `&& swift(<5.3)` instead of being deleted?

Reply:
that's reason why I created the type alias, so the syntax for swift <5.3  and  swift >=5.3 is the same